process/templates/WG_NAME_incubating_stage.md

```diff
@@ -1,21 +1,36 @@
 ## Working Group incubation application
 
-### List of regular contributors
-The WG must have a minimum of 5 contributors from at least 3 different organizations attending regulary.
+### List WG Chair(s) and or Vice Chair
+The WG must have a minimum of 1 Chair
 
   * "name, affiliation, GitHub ID"
 
-### TAC sponsor
-The WG must have a TAC sponsor:
 
+### Working Group (WG) has met all Sandbox requirement
+  * "link to sandbox PR"
```

Comment:
Same comment as before: I think there is no need to ask people to dig this up.

Reply:
i'll add in "if exists", ty